src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.enums.Color;
+
+public class Rectangle extends Figure {
+    private int sideOne;
+    private int sideTwo;
```

Comment:
naming variables with digits - it's a bad practice, consider naming like 'width' and 'height'